controller/search.js

```diff
@@ -21,6 +22,7 @@ function setup( apiConfig, esclient, query, should_execute ){
     const debugLog = new Debug('controller:search');
 
     let cleanOutput = _.cloneDeep(req.clean);
+
```

Comment:
Same for whitespace. Ideally there'd be no extra whitespace changes not part of the actual code changes. Not a very big deal of course :P

Reply:
@orangejulius Done!